src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -453,6 +456,13 @@ def _run_container_on_free_port(self, config: ContainerConfig) -> Tuple[Containe
                     raise err
         raise RetryCountExceeded(f"Unable to start container after {CONTAINER_START_RETRY_COUNT} retries")
 
+    def add_volume(self, name: str, config: ContainerConfig) -> Volume:
```

Comment:
Do we really want this method to be public?

Reply:
Actually we can move its call to `run_adcm_from_config` and use `remove_container_volumes` in fixture's teardown instead of using return value of the method `add_volume`.

Reply:
Done.